lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -178,19 +186,25 @@ def get_postgresql_version(self) -> str:
             logger.error(f"Failed to get PostgreSQL version: {e}")
             raise PostgreSQLGetPostgreSQLVersionError()
 
-    def is_tls_enabled(self) -> bool:
+    def is_tls_enabled(self, check_current_host: bool = False) -> bool:
```

Comment:
Maybe it's not appropriate, so feel free to ignore.

Is it valuable to, instead of taking `check_current_host`, instead to take a host address?
That way you could arbitrarily check address for TLS, or pass `self.charm.unit.private_address` or whatever. Not sure if that's useful/appropriate for this use-case though.

Reply:
In this case, I don't see the need for checking in that way. It seems to be good, if we find a use case that needs that. Probably in the future this method may be removed to a better check inside the charm code.